src/algorithms/calorimetry/CalorimeterClusterRecoCoG.cc

```diff
@@ -91,14 +91,6 @@ namespace eicrecon {
           }
         );
 
-        // FIXME: The code below fails for HcalEndcapPClusters. This does not happen for
-        // FIXME: all calorimeters. A brief scan of the code suggests this could be caused
-        // FIXME: by the CalorimeterHitDigi algorithm modifying the cellID for the raw hits.
-        // FIXME: Thus, the cellID values passed on through to here no longer match those
-        // FIXME: in the low-level truth hits. It likely works for other detectors because
-        // FIXME: their u_fields and u_refs members are left empty which effectively results
-        // FIXME: in the cellID being unchanged.
-
         // 2. find mchit with same CellID
         // find_if not working, https://github.com/AIDASoft/podio/pull/273
         //auto mchit = std::find_if(
```

Comment:
How about we uncomment the code and remove the loop version to show that this is indeed would be a no-op change?

Reply:
Well, std::find_if still isn't working (the PODIO issue linked to still isn't resolved), but the loop version is...

Reply:
Okay, I see now. The comment was added in 46b23c42efc5fb3f977eec5e501c173aa88d7dd7 during the porting effort. There are some tricks that Digi/Reco can do to the cellID's, so we are not completely out of the water, but storing issues in the FIXME's is not the way in any case.